### PR TITLE
feat: add Polygon, Base mainnet and test in the network list

### PR DESCRIPTION
**Pull request by lumtis**
Closes: https://github.com/zeta-chain/networks/issues/53

## Summary by CodeRabbit

- **New Features**
	- Expanded support for multiple blockchain networks, including base_mainnet, base_testnet, and polygon_mainnet.
	- Enhanced configurations include detailed attributes such as chain IDs, fee structures, asset definitions, and API endpoints for improved network integration.

### CODE REVIEW

File: data/networks.json [hunk 1]
```diff
@@ -1,4 +1,158 @@
...
+          {
+            "denom": "eth",
+            "exponent": 18
+          }
+        ],
+        "base": "eth",
+        "symbol": "ETH"
+      }
+    ],
+    "api": [
```
Comment (fadeev) [inline data/networks.json:89]:
```suggestion
    "apps": [
      {
        "type": "explorer",
        "url": "https://sepolia.basescan.org/",
        "tx": "/tx/${tx}",
        "address": "/address/${address}"
      }
    ],
    "api": [
```

File: data/networks.json [hunk 2]
```diff
@@ -1,4 +1,158 @@
...
```
Comment (fadeev) [inline data/networks.json:38]:
```suggestion
    "apps": [
      {
        "type": "explorer",
        "url": "https://basescan.org/",
        "tx": "/tx/${tx}",
        "address": "/address/${address}"
      }
    ],
    "api": [
```

**Comment by fadeev** [comment]
I would add explorers, but other than that, looks good.

**Comment by lumtis** [comment]
CI error looks unrelated to the change

**Review by fadeev — APPROVED** [review]

**Pull request merged**